e2e/upgrade-cephfs.go

```diff
@@ -234,6 +234,16 @@ var _ = Describe("CephFS Upgrade Testing", func() {
 				}
 				deployCephfsPlugin()
 
+				err = waitForDeploymentComplete(cephfsDeploymentName, cephCSINamespace, f.ClientSet, deployTimeout)
```

Comment:
check is present here https://github.com/ceph/ceph-csi/blob/devel/e2e/upgrade-cephfs.go#L147-L158?

Reply:
> check is present here https://github.com/ceph/ceph-csi/blob/devel/e2e/upgrade-cephfs.go#L147-L158?

This check is for older cephcsi version in upgrade tests. 

The wait I've added is for after deletion and cephcsi image is upgraded to the present changes.